micro-benchmarks/DRB140-reduction-barrier-orig-yes.c

```diff
@@ -8,9 +8,9 @@
  */
 
 
-/* The initialization of a=0@25:5 is not synchronized with the update of a@29:7 as a result of the
+/* The assignment to a@25:7 is  not synchronized with the update of a@29:11 as a result of the
  * reduction computation in the for loop.
- * Data Race pair: a@25:5 and a@29:7
+ * Data Race pair: a@29:7 and a@29:11
```

Comment:
```suggestion
 * Data Race pair: a@25:5 and a@27:33
```
All accesses inside of the worksharing-loop region are to the private instance of a. The possible race (depending on the implementation of the reduction) is between the initialization and the reduction of the private instances of a.